app/views/search/partials/firm/_adviser_distance.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="firm__adviser-distance t-distance_to_the_nearest_adviser">
+  <%= svg_icon :map_pin, class: 'firm__adviser-distance-pin' %>
+  <%= t('search.result.adviser') %>
+  <%= t('search.result.miles_away', distance: firm.closest_adviser.to_f.round(2)) %>
```

Comment:
This feels hacky. I don't think we should be reusing the generated string from FirmResult#closest_adviser.